### PR TITLE
Add Coffee Card component

### PR DESCRIPTION
**Pull request by clucasalcantara**
### Summary

Related to #5 

This PR aims to solves #5  adding the coffee card component. Please notice that all information is static and should be updated into a next phase.

### Preview
screenshot 2019-01-25 at 07 51 25

### CODE REVIEW

File: src/components/atoms/extra-info/index.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import ExtraInfo from './ExtraInfo';
+
+export default ExtraInfo
```
Comment (rafaelverger) [inline src/components/atoms/extra-info/index.js]: Why do u need to implement the component in another file? Why dont u just do it in index.js?

File: src/components/atoms/rating/index.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import Rating from './Rating'
+
+export default Rating
```
Comment (rafaelverger) [inline src/components/atoms/rating/index.js]: same

File: src/components/molecules/card/Card.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import { Card as CardContainer, Image } from 'semantic-ui-react'
+// UI Elements
+import { ExtraInfo, Rating } from 'components/atoms'
+
+const { Content, Header, Meta } = CardContainer
+
+const Card = () => (
+    <CardContainer>
+      <Image src="https://media-cdn.tripadvisor.com/media/photo-s/07/38/89/5c/seven-cafe.jpg" />
+        <Content>
```
Comment (rafaelverger) [inline src/components/molecules/card/Card.js]: wrong indentation

File: src/components/atoms/extra-info/ExtraInfo.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const ExtraInfo = () => (
+    <div className="extra content">
+        <i aria-hidden="true" className="wifi icon"></i> Bom · $$$ · Tranquilo
```
Comment (rafaelverger) [inline src/components/atoms/extra-info/ExtraInfo.js]: You could create it parametrized already. Also `ExtraInfo` is not quite semantic...  something like `EstablishmentDetails` would be better, don't u think?

File: src/components/atoms/rating/Rating.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+const Rating = () => (
+    <div className="ui disabled rating" role="radiogroup" tabindex="0">
+        <i tabindex="-1" aria-checked="false" aria-posinset="1" aria-setsize="5" className="active icon" role="radio"></i>
+        <i tabindex="-1" aria-checked="false" aria-posinset="2" aria-setsize="5" className="active icon" role="radio"></i>
+        <i tabindex="-1" aria-checked="true" aria-posinset="3" aria-setsize="5" className="active icon" role="radio"></i>
```
Comment (rafaelverger) [inline src/components/atoms/rating/Rating.js]: this should be a props

File: src/components/molecules/card/index.js [hunk 6]
```diff
@@ -0,0 +1,3 @@
+import Card from './Card'
+
+export default Card
```
Comment (rafaelverger) [inline src/components/molecules/card/index.js]: same question about not implementing in index

File: src/components/atoms/extra-info/ExtraInfo.js (hunk 4 above)
Comment (clucasalcantara) [reply src/components/atoms/extra-info/ExtraInfo.js]: My first idea about extra is to be a component wrapper and render content inside ExtraInfo section, but I didn't let this clear in my intentions xD now, speaking of it what do you think about it?

File: src/components/atoms/rating/Rating.js (hunk 5 above)
Comment (clucasalcantara) [reply src/components/atoms/rating/Rating.js]: Agree with you, the rating render logic must be improved (this repetition of i is just, blergh)

File: src/components/molecules/card/index.js (hunk 6 above)
Comment (clucasalcantara) [reply src/components/molecules/card/index.js]: already answered

File: src/components/molecules/card/Card.js (hunk 3 above)
Comment (clucasalcantara) [reply src/components/molecules/card/Card.js]: solved

**Pull request merged**